udata/harvest/tests/test_dcat_backend.py

```diff
@@ -551,6 +548,15 @@ def test_xml_catalog(self, rmock):
         assert resource_3.url == "http://data.test.org/datasets/1/resources/3"
         assert resource_3.type == "other"
 
+        # test dct:rights -> license support from dataset
+        dataset = Dataset.objects.get(harvest__dct_identifier="2")
+        assert dataset.license.id == "lov2"
+        assert dataset.extras["dcat"]["rights"] == ["Licence Ouverte Version 2.0"]
+
+        # test dct:rights storage in resource
```

Comment:
Also test on `accessRights`? (won't work with the `catalog.xml` fixture though...)

Reply:
Tested twice with geo_ide and geo2france, eg https://github.com/ecolabdata/udata/pull/2/files#diff-1b5b95b3634e50b14431b3ff7a286e38396acd7db2017b1626272ea60d3633dbR902

Reply:
The existing tests exercise "bubbling up" the resource `accessRights`, but I don't see a test that exercise "not bubbling up" like here?

Reply:
Right, I added a test for that https://github.com/ecolabdata/udata/pull/2/commits/c076f1a11fd1259b0a504e394b1e41400752b8ee#diff-1b5b95b3634e50b14431b3ff7a286e38396acd7db2017b1626272ea60d3633db